### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 2

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#328] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152324272-1735fd6b-155f-4e78-bee9-b20e9521d6ca.png)

**Comment by danieldinella** [comment]
1. Per definizione A∨B = 1 solo se o A o B o entrambi sono uguali a 1. Ciò implica che o A o B o entrambe sono conseguenza logica di T. Dunque la risposta è vero.

**Comment by Elia-Belli** [comment]
- $1$ **Vera**: riscrivendola con solo connettivi logici e facendo la tavola della verità (tutti 1) si nota che $(T \land \lnot A \land \lnot B) \lor \lnot T \lor A \lor B \in TAUT$, quindi la 1 è vera.

- $2$ **Vera**: Sia $T \models (A \rightarrow B) = T \models (\lnot A \lor B) \in SAT$, allora sappiamo che $B=1 \lor A = 0$.
$T \cup {B} \in UNSAT$ ovvero $T \land B \in UNSAT$ implica che $B=0$, allora per mantenere $SAT$ l'ipotesi $A = 0$.
$T \cup {A} \in UNSAT$ ovvero $T \land A \in UNSAT$, sappiamo che $A=0$ allora $T \cup {A} \in UNSAT$, quindi la 2 è vera.

- $3$ **Falsa**: Sia $T \models (A \rightarrow B) = T \models (\lnot A \lor B) \in SAT$, allora sappiamo che $B=1 \lor A = 0$.
$T \cup {A} \in UNSAT$ ovvero $T \land A \in UNSAT$ implica che $A=0$, il che rispetta l'ipotesi.
$T \cup {B} \in UNSAT$ ovvero $T \land B \in UNSAT$, sia che $B =1 \lor B =0$ l'ipotesi è rispettata grazie ad $A =0$, ma se $B =1$ $T \cup {B} \in SAT$, quindi la 3 è falsa.

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/217858920-ffaedd6e-169e-481c-bbd6-2bc3f8a9beb2.png)
